src/org/labkey/targetedms/parser/SkylineDocumentParser.java

```diff
@@ -2202,27 +2202,27 @@ private void readGeneralTransition(XMLStreamReader reader, GeneralTransition tra
 
         String sLens = reader.getAttributeValue(null, "s_lens");
```

Comment:
Should be changed to "explicit_s_lens".  This is the new name of the attribute.  It used to be "s_lens" when this attribute was still associated with general_transition_group_type (precursor).  This info is in the Word doc.

Reply:
Thanks. I didn't read far enough!